ui/src/components/helpers/SermonNotes.js

```diff
@@ -19,6 +19,35 @@ export const getBiblePassage = async (passage, currentText) => {
     }
     const [chapter, verse] = chapterAndVerse.split(':');
 
+    // handle if its different chapter
+    const isDifferentChapter = chapterAndVerse.split(':').length > 2;
+    if (bible && bible[book] && isDifferentChapter) {
+      const [firstChapterVerse, lastChapterVerse] = chapterAndVerse.split('-');
+      const [firstChapter, firstVerse] = firstChapterVerse.split(':');
+      const [lastChapter, lastVerse] = lastChapterVerse.split(':');
+      let passageBlock = '';
+      for (let i = Number(firstChapter); i <= lastChapter; i++) {
```

Comment:
perhaps can rename i and j for readability
```suggestion
      for (let currentChapter = Number(firstChapterNum); currentChapter <= Number(lastChapterNum); currentChapter++) {
    if (currentChapter === Number(firstChapterNum)) {
      for (let currentVerse = Number(firstVerseNum); bible[book][currentChapter][currentVerse] !== undefined; currentVerse++) {
        passageBlock += `${currentVerse} ${bible[book][currentChapter][currentVerse]} `;
      }
      passageBlock += '<br />';
    } else if (currentChapter === Number(lastChapterNum)) {
      passageBlock += '<br />';
      for (let currentVerse = 1; currentVerse <= Number(lastVerseNum); currentVerse++) {
        passageBlock += `${currentVerse} ${bible[book][currentChapter][currentVerse]} `;
      }
    } else {
      passageBlock += '<br />';
      for (let currentVerse = 1; bible[book][currentChapter][currentVerse] !== undefined; currentVerse++) {
        passageBlock += `${currentVerse} ${bible[book][currentChapter][currentVerse]} `;
      }
      passageBlock += '<br />';
    }
  }
```